corehq/apps/hqwebapp/views.py

```diff
@@ -395,6 +396,24 @@ def debug_notify(request):
             "If you want to achieve a 500-style email-out but don't want the user to see a 500, use notify_exception(request[, message])")
     return HttpResponse("Email should have been sent")
 
+
+@require_POST
+def jserror(request):
+    stack = request.POST.get('stack', None)
+
+    ExponentialBackoff.increment(stack)
+    if not ExponentialBackoff.should_backoff(stack):
+        notify_js_exception(
+            request,
+            message=request.POST.get('message', None),
+            stack=stack,
+            line=request.POST.get('line', None),
+            filename=request.POST.get('filename', None),
+        )
```

Comment:
good idea about the stack trace N lines. i'll do that. i also like the blowing up factor. i'll incorporate that as well